particles/Tutorial/Javascript/Demo/source/GetPlayers.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+ /* global defineParticle */
+
+ defineParticle(({DomParticle, html}) => {
+
+    const template = html`
+  <b>Please enter your name:</b>
+  <input value="{{name}}" placeholder="Enter your name" spellcheck="false" on-change="onNameInputChange"> <button on-click="onClick">Enter</button>
+    `;
+
+  return class extends DomParticle {
+    get template() {
+      return template;
+    }
+
+    update(props, {enteredName}) {
+      // Set default name.
```

Comment:
I don't think some of these comments add value.

This one and: // Save entered name.

WDYT?